README.md

```diff
@@ -182,6 +185,38 @@ to get time averages for main detection function and strategies for the whole sa
 if you want see measures by sample file
 
 
+
+.gitattributes
+--------------
+
+Like in linguist you can override the strategies via `.gitattributes` file.
+Add a `.gitattributes` file to the directory and use the same matchers that you would uses in linguist `linguist-documentation`,`linguist-language` or `linguist-vendored` to do the override.
+
+#### Vendored code
+
+Use the `linguist-vendored` attribute to vendor or un-vendor paths.
+
+```
+$ cat .gitattributes
+this-is-a-vendor-directory/ linguist-vendored
+this-is-not/ linguist-vendored=false
+```
+#### Documentation
+
+Documentation works the same way as vendored code but using `linguist-documentation` and `linguist-documentation=false`.
+
+#### Language assignation
+
+If you want some files to be classified according to certain language use `linguist-language=[language]`.
+
+```
+$ cat .gitattributes
+.*\.go linguist-language=MyFavouriteLanguage
```

Comment:
If the regex must be compatible with Golang regexp, does it mean that a .gitattribute file used in linguist has a different syntax? Does it make .gitattributes files for enry incompatibles with linguist ones?